src/utils/html.ts

```
@@ -0,0 +1,133 @@
+const allowedAttributes: Record<string, string[]> = {
+  a: ["href", "rel", "target"]
+};
+
+const purifierCache = new WeakMap<Window, DOMPurifyI>();
+
+function ensureEnv(): { window: Window & typeof globalThis; document: Document } {
+  if (typeof window !== "undefined" && typeof document !== "undefined") {
+    return { window: window as Window & typeof globalThis, document };
+  }
+
+  const jsdom = (eval("require")("jsdom") as typeof import("jsdom")).JSDOM;
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)